ext/http/http_next.rs

```diff
@@ -283,11 +305,31 @@ pub fn op_http_get_request_headers(slab_id: SlabId) -> Vec<ByteString> {
   // TODO(mmastrac): This should probably happen on the JS side on-demand
   if let Some(cookies) = cookies {
     let cookie_sep = "; ".as_bytes();
-    vec.push(ByteString::from(COOKIE.as_str()));
-    vec.push(ByteString::from(cookies.join(cookie_sep)));
+
+    vec.push(
```

Comment:
As this is constant, it can be `new_external_onebyte_static`:

https://docs.rs/rusty_v8/latest/rusty_v8/struct.String.html#method.new_external_onebyte_static